src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -50,6 +50,21 @@ internal bool AutomaticRedirection
             }
         }
 
+        internal ClientCertificateOption ClientCertificateOptions
+        {
+            get
+            {
+                return ClientCertificateOption.Manual;
+            }
+            set
+            {
+                if (ClientCertificateOption.Manual != value)
+                {
+                    throw PlatformNotSupportedException("libcurl does not support automatic client certificate selection");
```

Comment:
This string needs to be added to a PLATFORM-SPECIFIC strings file.  It shouldn't be added to the main strings.resx file unless it is "ifdef unix".  I'm not sure how to put `ifdef` in the resource file.  Maybe @ericstj knows the answer.